packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialog.js

```diff
@@ -93,6 +96,13 @@ const EnlargedDialogComponent = ({
   // Regardless of the drillDown level, we pass the base view content through
   const baseViewContent = contentByLevel?.[0]?.viewContent;
   const drillDownContent = contentByLevel?.[drillDownState.drillDownLevel]?.viewContent;
+  const baseViewConfig = contentByLevel?.[0]?.viewConfig;
+  const drillDownConfig = contentByLevel?.[drillDownState.drillDownLevel]?.viewConfig;
```

Comment:
I'm not entirely sure how `contentByLevel` is being calculated? Does that need to be updated for the new way drill downs are laid out?

Reply:
Yep so it stores the viewConfig/viewContent of each level of drill down.